docs/guides/using-watt-with-node-config.md

```diff
@@ -0,0 +1,133 @@
+# Using Watt With Node Config
+
+[Node-config](https://www.npmjs.com/package/config) is a powerful configuration management package that helps organize settings across different deployment environments in your application. It creates a unified configuration system that works seamlessly with both [Watt](https://platformatic.dev/docs/watt/overview) and other `npm` modules.
+
+## Benefits of node-config
+
+[Node-config](https://www.npmjs.com/package/config) provides a flexible hierarchy for your application settings. You can define default values and then customize them for specific environments like development, QA, staging, and production. The configuration values can come from multiple sources, including configuration files, environment variables, command line parameters, and external services.
+
+## Installation and Setup
```

Comment:
The use case is actually different. We want to have each service in a watt app have their own node-config setup, we want to support potentially different environment name in different services.

This is achievable by setting env variables specific for each service.

To achieve this, you'd need to set (per service) [`NODE_CONFIG_DIR`](https://github.com/node-config/node-config/wiki/Environment-Variables#node_config_dir) and set `NODE_ENV` accordingly.

This is possible via https://platformatic.dev/docs/watt/configuration#services or https://platformatic.dev/docs/watt/configuration#web.